tests/tests.py

```diff
@@ -164,14 +167,15 @@ def test_expired_tokens_deleted(self):
 
     def test_update_token_key(self):
         self.assertEqual(AuthToken.objects.count(), 0)
-        token = AuthToken.objects.create(self.user)
+        instance, token = AuthToken.objects.create(self.user)
         rf = APIRequestFactory()
         request = rf.get('/')
         request.META = {'HTTP_AUTHORIZATION': 'Token {}'.format(token)}
         (self.user, auth_token) = TokenAuthentication().authenticate(request)
         self.assertEqual(
             token[:CONSTANTS.TOKEN_KEY_LENGTH],
-            auth_token.token_key)
+            auth_token.token_key
```

Comment:
Nitpick alert: I would expect a comma at the end of the line.

Reply:
Just to be clear, like this?
```
        self.assertEqual(
            token[:CONSTANTS.TOKEN_KEY_LENGTH],
            auth_token.token_key,
        )
```

Reply:
Exactly :-)